### PR TITLE
Update recipes with mysql based doctrine docker compose file

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | replaces #144
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Update recipes with mysql based doctrine docker compose file.

#### Why?

Else symfony will create a postgresql based docker compose file.

#### Possible Solutions

- **A: Use PostgreSQL Docker compose file** [#144](https://github.com/sulu/skeleton/pull/144)
- **B: Use Mysql Docker compose file**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-ChangeMe}
+        volumes:
+            - db-data:/var/lib/mysql
+            # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!
+            # - ./docker/db/data:/var/lib/mysql:rw
+###< doctrine/doctrine-bundle ###
+
+volumes:
+###> doctrine/doctrine-bundle ###
+    db-data:
```
Comment (alexander-schranz) [inline docker-compose.yml:19]: I didn't run this. @wachterjohannes could you test if this is all fine?
Config is based postgresql config provided by the recipe: https://github.com/sulu/skeleton/pull/144/files

Comment (wachterjohannes) [reply docker-compose.yml:19]: @alexander-schranz looks good had tested this a few days ago with a fresh installation. but would you go with postgre or mysql?

**Review by wachterjohannes — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (alexander-schranz) [reply docker-compose.yml:19]: I would say our community is more on mysql. /cc @chirimoya

**Comment by alexander-schranz** [comment]
As discussed we will wait here for feedback from the community, if we go with **postgresql**  or **mysql**.

**Comment by BaptisteCrouzet** [comment]
As seen on many benchmarks, like this one https://itnext.io/benchmark-databases-in-docker-mysql-postgresql-sql-server-7b129368eed7 PostgreSQL isn't a bad default choice when looking at read/write/update time, often really faster than MySQL.
I'm used to MySQL but I would be okay to switch on Postgre  🙂

**Comment by sanhofman** [comment]
We use MySQL as our go-to database, but lately receive more requests for PostgreSQL.

Are there benchmarks for Sulu with PostgreSQL vs MySQL? Could be interesting.

But overall, definitely fine to switch to PostgreSQL as default.

**Comment by alexander-schranz** [comment]
We sadly do not yet have any benchmarks yes about postgresql. If somebody is using postgresql, would be great if they could give us some stats about it to performance to mysql 8 with phpcr.

We only did get once feedback when somebody did upgrade form PostgrSQL form 9.6 to 12 that the contact list which was just a indexed join was getting really slow. But we could not find out why and I'm not sure if that does still exist as it was a really early version of postgresql 12 at that time.

**Comment by alexander-schranz** [comment]
From our Partner [connectiv](https://sulu.io/partner/find-a-partner#connectiv-esolutions-gmbh) we got the response that they would prefer `MySQL` as default in Sulu.

**Comment by alexander-schranz** [comment]
A `third` option would be disable `docker` support by default in Sulu Skeleton by setting the `extra` -> `symfony` -> `docker` config to `false`: https://github.com/symfony/flex/pull/821

**Comment by niklasnatter** [comment]
We decided to include a `docker-compose.yml` file with a `mysql` database service  for the moment. The reason is that we do not have enough experience with PostgreSQL to include this as default choice confidently at the moment.
Still, we want to fully support PostgreSQL and we are open to change the default choice in the future 🙂

**Review by niklasnatter — APPROVED** [review]

**Comment by alexander-schranz** [comment]
Please see this discussion not as closed. We are still searching here for feedback. If somebody have experience performance on PHPCR based content mysql vs postgresql would be great.

**Pull request merged**